specification/securityinsights/resource-manager/Microsoft.SecurityInsights/stable/2024-03-01/AutomationRules.json

```diff
@@ -942,17 +1220,47 @@
             "value": "Property",
             "description": "Evaluate an object property value"
           },
+          {
+            "value": "PropertyArray",
+            "description": "Evaluate an object array property value"
+          },
           {
             "value": "PropertyChanged",
             "description": "Evaluate an object property changed value"
           },
           {
             "value": "PropertyArrayChanged",
             "description": "Evaluate an object array property changed value"
+          },
+          {
+            "value": "Boolean",
+            "description": "Apply a boolean operator (e.g AND, OR) to conditions"
           }
         ]
       }
     },
+    "EntityManualTriggerRequestBody": {
+      "description": "Describes the request body for triggering a playbook on an entity.",
+      "required": [
+        "logicAppsResourceId"
+      ],
+      "type": "object",
+      "properties": {
+        "incidentArmId": {
+          "description": "The incident id to associate the entity with.",
+          "type": "string"
+        },
+        "tenantId": {
+          "format": "uuid",
+          "description": "The tenant id of the playbook resource.",
+          "type": "string"
+        },
+        "logicAppsResourceId": {
+          "description": "The resource id of the playbook resource.",
+          "type": "string"
```

Comment:
>type [](http://example.com/codeflow?start=11&length=4)

You can also annotate it with `"format": "arm-id"`

Reply:
And perhaps [https://azure.github.io/autorest/extensions/#x-ms-arm-id-details](https://azure.github.io/autorest/extensions/#x-ms-arm-id-details)